### ISSUE TITLE
Minimal config file fails to load

### ISSUE THREAD

**Issue by HEdingfield** [BrightSpots/rcv#318]
I thought this was addressed in #296, but, when attempting to load a stripped-down config file with the bare minimum of "required" fields (see an example below), it doesn't properly load for some reason and the error message is entirely unhelpful:

> 2019-06-28 10:22:07 PDT INFO: Successfully loaded contest config: C:\blah\rcv\src\test\resources\network\brightspots\rcv\test_data\sample_interactive_tiebreak\sample_interactive_tiebreak_config.json
> 2019-06-28 10:22:07 PDT SEVERE: Error processing candidate data:
> java.lang.NullPointerException

```
{
  "outputSettings": {
    "contestName": "Sample Interactive Tiebreak",
    "outputDirectory": "output"
  },
  "cvrFileSources": [
    {
      "filePath": "sample_interactive_tiebreak_cvr.xlsx",
      "firstVoteColumnIndex": 2,
      "firstVoteRowIndex": 2
    }
  ],
  "candidates": [
    {
      "name": "Mookie Blaylock"
    },
    {
      "name": "Yinka Dare"
    },
    {
      "name": "George Gervin"
    },
    {
      "name": "Sedale Threatt"
    }
  ],
  "rules": {
    "tiebreakMode": "interactive",
    "overvoteRule": "alwaysSkipToNextRank",
    "maxRankingsAllowed": 3,
    "maxSkippedRanksAllowed": 0,
    "numberOfWinners": 1,
    "decimalPlacesForVoteArithmetic": 4,
    "minimumVoteThreshold": 0
  }
}

```

Assigning to all of us for now, but if you're going to start working on it, please get rid of the other assignees.

**Comment by HEdingfield** [comment]
This is triggered by the following line in `ContestConfig.processCandidateData()`:

`if (source.getProvider().equals("CDF")) {`

I'll submit a fix.

**Comment by HEdingfield** [comment]
Closed with #319.
